### ISSUE TITLE
Poloniex async Type error

### ISSUE THREAD

**Issue by nmiculinic** [ccxt/ccxt#461]
```
  File "/usr/local/lib/python3.6/site-packages/ccxt/async/poloniex.py", line 428, in fetch_open_orders
    orders = await self.fetch_orders(symbol, params)

  File "/usr/local/lib/python3.6/asyncio/coroutines.py", line 109, in __next__
    return self.gen.send(None)

  File "/usr/local/lib/python3.6/site-packages/ccxt/async/poloniex.py", line 372, in fetch_orders
    market = self.market(symbol)

  File "/usr/local/lib/python3.6/site-packages/ccxt/base/exchange.py", line 803, in market
    raise ExchangeError(self.id + ' does not have market symbol ' + symbol)

TypeError: must be str, not NoneType
```

ccxt==1.9.341

**Comment by kroitor** [comment]
Hi! Poloniex fetchOpenOrders requires a symbol (it is also true for many other exchanges as well). So, in a generic case, we need to pass the symbol to that method.

**Comment by kroitor** [comment]
Fixed in 1.9.362+. Thx again!
